package.json

```diff
@@ -73,7 +73,7 @@
     "@patternfly/react-core": "^4.276.6",
     "@patternfly/react-icons": "^4.93.6",
     "@patternfly/react-table": "^4.113.0",
-    "@patternfly/react-tokens": "^4.94.6",
+    "@patternfly/react-tokens": "^5.1.2",
```

Comment:
The v5 package may conflict with the v4 packages and we should refrain from mixing them.